recipe/build.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/bash
 set -ex
 
-if [[ ${gpu_variant:-} = "cuda-12" ]]; then
+if [[ ${gpu_variant:0:5} = "cuda-" ]]; then
     CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_CUDA_ARCHITECTURES=all"
-    LLAMA_ARGS="${LLAMA_ARGS} -DLLAMA_CUBLAS=ON"
-    export LDFLAGS="$LDFLAGS -Wl,-rpath-link,/usr/lib64"
-elif [[ ${gpu_variant:-} = "cuda-11" ]]; then
-    CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_CUDA_ARCHITECTURES=all"
-    LLAMA_ARGS="${LLAMA_ARGS} -DLLAMA_CUBLAS=ON"
-    export CUDACXX=/usr/local/cuda/bin/nvcc
-    export CUDAHOSTCXX="${CXX}"
+    LLAMA_ARGS="${LLAMA_ARGS} -DLLAMA_CUDA=ON"
+    if [[ ${gpu_variant:-} = "cuda-11" ]]; then
+        export CUDACXX=/usr/local/cuda/bin/nvcc
+        export CUDAHOSTCXX="${CXX}"
+    else
+        LDFLAGS="$LDFLAGS -Wl,-rpath-link,${PREFIX}/lib/stubs/"
+        ln -s ${PREFIX}/lib/stubs/libcuda.so ${PREFIX}/lib/stubs/libcuda.so.1
```

Comment:
`libcuda.so.1` seems to be provided by https://github.com/conda-forge/cuda-compat-feedstock (more particularly, the `cudao-compat` packages (https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Fcuda-compat-impl-525.60.13-h59595ed_0.conda). SHould we rely on these packages instead of symlinking here?

Reply:
Here is how it's used in conda-forge: https://github.com/conda-forge/bmf-feedstock/blob/cd31d557537a4b16e471d98510bc915754b7c4b1/recipe/meta.yaml#L66-L71

Reply:
Thanks, I didn't spot that! how did you find that? github search for libcuda.so.1?

Reply:
I used https://conda-metadata-app.streamlit.app/Search_by_file_path.